### DISCUSSION TITLE
Need help: Haven't been updated with the Github education benefits after 5 days, maybe because I am out of clg campus how can I reapply

### DISCUSSION THREAD

**Discussion by Harshstag** [community/community#132755] [GitHub Education]
### Hi everyone,

Need help: Haven't been updated with the Github education benefits after 5 days, maybe because I am out of clg campus now how can I reapply

**Comment by davevad93** [comment]
Hi @Harshstag ,

If your application status says that your benefits will be available soon, it means that you've been approved and you just have to wait for your benefits to become available, you don't need to reapply. When they become available, the status label will turn purple and change to:

`Congrats! Your request for @Harshstag was approved and your benefits are now available.`

That could take a while, usually it's a matter of a few days but sometimes it can take even a couple of weeks, the waiting time may vary depending on different factors e.g. the volume of applications.
